html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
+      import { 
+        fullActivityDescription, 
+        shortActivityDescription, 
+        libraryModelList, 
+        cclDiseaseActivityUrl 
+      } from '/assets/data/host.js';
+
+      document.addEventListener("DOMContentLoaded", () => {
+        // Host page content init
+        document.getElementById("activity-text-full").innerHTML = fullActivityDescription;
+        document.getElementById("activity-text-short").innerHTML = shortActivityDescription;
+        document.getElementById("library-model").innerHTML = libraryModelList;
+        document.getElementById("activity-img").src = cclDiseaseActivityUrl;
+
+        // Open & close modal functionality
+        const readMoreBtn = document.getElementById("read-more-btn");
+        const closeModalBtn = document.getElementById("close-modal-btn");
+        const modalContainer = document.getElementById("modal-container");
+
+        readMoreBtn.onclick = () => {
```

Comment:
When you move this into the JS, you can use the `byEID` function as shorthand for `document.getElementById`.

So:

```javascript
my_big_line;

modalContainer.onclick = () => {
```

can become:

```javascript
byEID("modal-container").onclick = () => {
```

Reply:
Done!